Update noParse docs (#4908)
webpack/webpack.js.org: src/content/configuration/module.mdx

Prevent webpack from parsing any files matching the given regular expression(s). Ignored files **should not** have calls to `import`, `require`, `define` or any other importing mechanism. This can boost build performance when ignoring large libraries.

> `noParse` can be also used as a way to deliberately prevent  expansion of all `import`, `require`, `define` etc. calls, if you're sure those calls are unreachable at runtime.

Reviewer: ```suggestion
`noParse` can be also used as a way to deliberately prevent expansion of all `import`, `require`, `define` etc. calls for cases when those calls are unreachable at runtime.
```

> For example, when building a project for browser target, and using a third-party library that was prebuilt as a dual browser/nodejs lib, and requires some nodejs built-ins, like `require('os')`.

Reviewer: ```suggestion
For example, when building a project for `'browser'` [target](/configuration/target/) and using a third-party library that was prebuilt for both browser and Node.js and it requires Node.js built-ins e.g. `require('os')`.
```

> Note the need to use `[\\/]` in regex to match `\` on Windows and `/` on Mac/Linux.

Reviewer: ```suggestion
T> You may need to use `[\\/]` in regex to match `\` on Windows and `/` on Mac/Linux.
```
